dbt_copilot_helper/commands/conduit.py

```diff
@@ -37,28 +37,68 @@ class CreateTaskTimeoutConduitError(ConduitError):
     pass
 
 
+class ParameterNotFoundConduitError(ConduitError):
+    pass
+
+
+class AddonNotFoundConduitError(ConduitError):
+    pass
+
+
 CONDUIT_DOCKER_IMAGE_LOCATION = "public.ecr.aws/uktrade/tunnel"
 CONDUIT_ADDON_TYPES = [
     "opensearch",
-    "postgres",
+    "rds-postgres",
+    "aurora-postgres",
     "redis",
 ]
 
 
-def normalise_string(to_normalise: str) -> str:
-    output = re.sub("[^0-9a-zA-Z]+", "-", to_normalise)
-    return output.lower()
+def normalise_secret_name(addon_name: str) -> str:
+    return addon_name.replace("-", "_").upper()
+
+
+def get_addon_type(app: Application, env: str, addon_name: str) -> str:
+    session = app.environments[env].session
+    ssm_client = session.client("ssm")
+    addon_type = None
+
+    try:
+        addon_config = json.loads(
+            ssm_client.get_parameter(
+                Name=f"/copilot/applications/{app.name}/environments/{env}/addons"
+            )["Parameter"]["Value"]
+        )
+    except ssm_client.exceptions.ParameterNotFound:
+        raise ParameterNotFoundConduitError
+
+    if addon_name not in addon_config.keys():
+        raise AddonNotFoundConduitError
+
+    for name, config in addon_config.items():
+        if name == addon_name:
+            addon_type = config["type"]
+
+    if not addon_type or addon_type not in CONDUIT_ADDON_TYPES:
+        raise InvalidAddonTypeConduitError(addon_type)
```

Comment:
Maybe in a subsequent PR, but how about producing an error that says something like: "Addon dwPostgres not found, use one of the following: dwRedis, dwOpensearch".